capstone/src/main/java/com/step902020/capstone/UniversityRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.step902020.capstone;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.cloud.gcp.data.datastore.repository.DatastoreRepository;
+
+public interface UniversityRepository extends DatastoreRepository<University, Long> {
+
+  public University findFirstByName(String name);
+  public Long deleteAllByName(String name);
```

Comment:
Why would you delete a university?

Reply:
It's for cleaning up the made-up university in the integration tests.